### DISCUSSION TITLE
Change z-index of el-loading-mask

### DISCUSSION THREAD

**Discussion by nhatimme** [element-plus/element-plus#11880] [General]
Hi, 

Element Plus v-loading is conflicting with the z-index of 2000. I want to bring it back to 100. What is the best way to do this? Ofcourse If I update vue version of Element Plus, it should not overwrite the settings again. Thanks.

**Comment by chenxch** [comment]
Can you provide a playground?

**Comment by nhatimme** [comment]
Well, we are working with a template which have Bootstrap modals. Those modals are using z-index: 100-199 range. We want to make sure the css of the v-loading element is not 2000 as z-index but 100. The question is: How can I overwrite the default css of this element? I don't want to change the core directly, as if we update Element Plus, it will overwrite it again.
